explorer/client/src/app/App.tsx

```diff
@@ -1,27 +1,49 @@
 // Copyright (c) 2022, Mysten Labs, Inc.
 // SPDX-License-Identifier: Apache-2.0
 
+import { QueryClient, QueryClientProvider } from '@tanstack/react-query';
```

Comment:
I like this addition of `react-query`. It looks like this will be a more straightforward replacement of our fetch/useEffect code that we should apply more broadly across the codebase at some point.